modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -150,6 +150,35 @@ The channels you need for this procedure are:
 
 |===
 
+ifeval::[{uyuni-content} == true]
+
+[[es-channels-cli]]
+[cols="1,1,1,1", options="header"]
+.ES Channels - CLI
+|===
+
+| OS Version
+| Base Channel
+| Client Channel
+| Tools Channel
+
+| {sll} 7
+| rhel-x86_64-server-7
```

Comment:
I wonder if it is`rhel-x86_64-server-7` or `res7-x86_64`.

See:

![image](https://user-images.githubusercontent.com/4226070/213225853-84030cf3-cdcc-4951-a3e5-f187f283f0a7.png)

Reply:
BTW, maybe this question is valid for SUSE Manager as well.

Reply:
I also could not find it yesterday.  @lucidd can you help?

Reply:
`"rhel-x86_64-server-7` is the base channel label. `res7-x86_64` is the updates channel if i see that correctly.

Reply:
we currently mention `rhel-x86_64-server-7` correctly as the base channel in the docs.